modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -0,0 +1,146 @@
++
+The CCM manifest are required for deploying Oracle’s CCM during cluster installation so that {product-title} can connect to the external {oci} platform. The CSI custom manifests are required for deploying Oracle’s CSI driver during cluster installation so that {product-title} can claim required objects from {oci}.
++
+[IMPORTANT]
+====
+You must modify the secret `oci-cloud-controller-manager` defined in the link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/manifests/oci-ccm.yml[`oci-ccm.yml`] configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
+====
+
+. Use the Agent-based Installer to generate a minimal ISO image, which excludes the `rootfs` image, by entering the following command in your {product-title} CLI. You can use this image later in the process to boot all your cluster’s nodes. 
++
+[source,terminal]
+----
+$ ./openshift-install agent create image --log-level debug
+----
++
+The previous command also completes the following actions:
++
+* Creates a subdirectory, `./<installation_directory>/auth directory:`, and places `kubeadmin-password` and `kubeconfig` files in the subdirectory.  
+* Creates a `rendezvousIP` file based on the IP address that you specified in the `agent-config.yaml` configuration file. 
+* Optional: Any modifications you made to `agent-config.yaml` and `install-config.yaml` configuration files get imported to the Zero Touch Provisioning (ZTP) custom resources. 
++
+[IMPORTANT]
+====
+The Agent-based Installer uses {op-system-first}. The `rootfs` image, which is mentioned in a subsequent listed item,  is required for booting, recovering, and repairing your operating system. 
+====
+
+. Configure the `agent-config.yaml` configuration file to meet your organization’s requirements. 
++
+.Example `agent-config.yaml` configuration file that sets values for an IPv4 formatted network.
+[source,yaml]
+----
+apiVersion: v1alpha1
+metadata:
+  name: <cluster_name> <1>
+  namespace: <cluster_namespace> <2>
+rendezvousIP:<ip_address_from_CIDR> <3>
+bootArtifactsBaseURL:<server_URL> <4>
+# …
+----
+<1> The cluster name that you specified in your DNS record. 
+<2> The name of your cluster on {product-title}. 
+<3> If you are using IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.  
+<4> The URL of the server where you want to upload the `rootfs` image.
++
+[IMPORTANT]
+====
+You might need to return to the {oci} web console and update your DNS zone's record details to match information that you specified in the `agent-config.yaml` configuration file. See the "Creating OCI infrastructure resources and services" section.
```

Comment:
this is not needed.

Reply:
Thanks, @pawanpinjarkar . Removed. Would you be OK to add a LGTM for the work that is based on the current workflow with Oracle (if the Terraform script becomes available we could revist the content.